doc/modules/mod_pubsub.md

```diff
@@ -118,6 +118,45 @@ This node type **requires** `publish-options` with at least `device_id` and `ser
 
 If you'd like to learn more about metrics in MongooseIM, please visit the [MongooseIM metrics](../operation-and-maintenance/Mongoose-metrics.md) page.
 
+#### Overall PubSub action metrics
+
+For every PubSub action, like node creation, subscription, publication there are following kind of metrics:
+
+* count - number of total and in last minute actions
+* errors - number of total and in last minute errors for given action
+* time - the time it took to finish the action
```

Comment:
I'd state explicitly that `count` and `errors` are spirals and `time` is histogram. With this, "total and in last minute" won't be necessary (which is not very clear here anyway), if someone is not familiar with metric types in MongooseIM.

Reply:
Also, I'd add a mention that `time` applies only to successful actions.

Reply:
👍